<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nodePalette</class>
 <widget class="QWidget" name="nodePalette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="2" column="4">
      <widget class="QToolButton" name="PolyShearU">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="AddNode">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="IndependentMove">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="SymMove">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="PolySplit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QToolButton" name="PolyShearR">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="AsymMove">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="MoveNode">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QToolButton" name="PolyShearD">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="EditControl">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="PolyShearL">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="PolyMirrorV">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="DeleteNode">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="PolyMirrorH">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="BezierClose">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QSpinBox" name="RotVal"/>
     </item>
     <item row="2" column="0">
      <widget class="ScrSpinBox" name="scaleDistance"/>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="Enlarge">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="RotateCW">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="Shrink">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="scalePercentage"/>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="RotateCCW">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="Reduce">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="Expand">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="1" column="1">
      <widget class="ScrSpinBox" name="XSpin"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="TextLabel1">
       <property name="text">
        <string>&amp;X:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="ScrSpinBox" name="YSpin"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="TextLabel2">
       <property name="text">
        <string>&amp;Y:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="PosOriginLabel">
         <property name="text">
          <string>Origin:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DropdownButton" name="PosOriginButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QToolButton" name="PreviewMode">
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="EditCont">
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ResetCont">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ResetContClip">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ResetShape2Clip">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="resetDefaultButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="editEditButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelEditButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">ui/scrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>DropdownButton</class>
   <extends>QToolButton</extends>
   <header>ui/widgets/dropdown_button.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>MoveNode</tabstop>
  <tabstop>EditControl</tabstop>
  <tabstop>AddNode</tabstop>
  <tabstop>DeleteNode</tabstop>
  <tabstop>PolySplit</tabstop>
  <tabstop>SymMove</tabstop>
  <tabstop>AsymMove</tabstop>
  <tabstop>IndependentMove</tabstop>
  <tabstop>BezierClose</tabstop>
  <tabstop>PolyMirrorH</tabstop>
  <tabstop>PolyMirrorV</tabstop>
  <tabstop>PolyShearL</tabstop>
  <tabstop>PolyShearR</tabstop>
  <tabstop>PolyShearU</tabstop>
  <tabstop>PolyShearD</tabstop>
  <tabstop>RotVal</tabstop>
  <tabstop>RotateCCW</tabstop>
  <tabstop>RotateCW</tabstop>
  <tabstop>scalePercentage</tabstop>
  <tabstop>Expand</tabstop>
  <tabstop>Shrink</tabstop>
  <tabstop>scaleDistance</tabstop>
  <tabstop>Enlarge</tabstop>
  <tabstop>Reduce</tabstop>
  <tabstop>PosOriginButton</tabstop>
  <tabstop>XSpin</tabstop>
  <tabstop>YSpin</tabstop>
  <tabstop>PreviewMode</tabstop>
  <tabstop>EditCont</tabstop>
  <tabstop>ResetCont</tabstop>
  <tabstop>ResetContClip</tabstop>
  <tabstop>ResetShape2Clip</tabstop>
  <tabstop>resetDefaultButton</tabstop>
  <tabstop>editEditButton</tabstop>
  <tabstop>cancelEditButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
